components/Dashboard/Input/FileInput.tsx

```diff
@@ -26,7 +26,7 @@ export const FileInput = forwardRef(
       >
         <button
           type="button"
-          className="py-0.5 px-2 font-medium rounded-lg bg-dashboard-gray-6"
+          className="py-0.5 px-2 font-medium rounded-lg bg-dashboard-gray-6 cursor-pointer"
```

Comment:
adding `cursor: pointer` here has no effect because it already applies to `button` tag, if you want to achieve the style then consider adding a overlay div above all elements inside and setting that to cursor pointer or just ignore it altogether 🤷🏻‍♂️

Reply:
thanks, I will remove this className